packages/atomic/src/components/insight/user-actions/atomic-insight-user-actions-session/atomic-insight-user-actions-session.pcss

```diff
@@ -0,0 +1,17 @@
+@import '../../../../global/global.pcss';
+
+.session-start-icon__container {
+  background-color: #fbb439;
+}
+
+.user-action__separator {
+  background-color: var(--atomic-neutral);
+}
+
+.case-creation-action__text {
+  color: var(--atomic-success);
+}
+
+.text-xxs {
+  font-size: 0.625rem;
+}
```

Comment:
I would assume that this is something that already exists somewhere else in Atomic?

Otherwise is there a reason why we're adding text smaller than ANYTHING we've ever had in Atomic?

Reply:
I'm following the font-sizes from the mockups a I didn't find the size I was looking for already in Atomic, I will double check if not found I will check with UX.